src/vehicle_display.cpp

```diff
@@ -478,3 +478,35 @@ void vehicle::print_fuel_indicator( const catacurses::window &win, const point &
         }
     }
 }
+
+void vehicle::print_speed_gauge( const catacurses::window &win, const point &p, int spacing )
+{
+    if( spacing < 0 ) {
+        throw std::invalid_argument( "received negative value for spacing" );
```

Comment:
A hard game crash here is a little harsh. Maybe just a debugmsg and use some value for spacing that won't cause big problems?

Reply:
Yeah, that's better.

Reply:
Although, it would trigger on each draw.

Reply:
```suggestion
        spacing = 0;
        DebugLog( D_WARNING, D_MAIN ) << "Speed gauge spacing is nagetive. Changing it to 0";

Reply:
A debuglog is a little too lenient to me. The game doesn't need to crash, but it should say something's wrong fairly obviously.